manifests/run.pp

```diff
@@ -517,9 +527,12 @@
 
     if $ensure == 'absent' {
       if $facts['os']['family'] == 'windows' {
+        $absent_stop_command = "${docker_command} stop --time=${stop_wait_time} ${sanitised_title}"
+        $absent_stop_onlyif = "${docker_command} inspect ${sanitised_title}"
```

Comment:
Both of these need breaking up.

Also refer to the above comment for the `--time` arg.

Reply:
Same as above. If not broken up, it presents serious issues. To be more precise, I think most of the unbroken arguments in run.pp were breaking windows acceptance tests when tried to modify them. I cannot really tell you why. I dont feel like there is any reason windows should not be able to handle a 
`[$docker_command, 'inspect', $sanitised_title]`

Reply:
It would be good to spend a little bit of time understanding why the failures are occurring. 

The commands wont reach the os as an array. They are stringified by the provider.